### PR TITLE
[WIP] Replace rspec with minitest

### PR DESCRIPTION
**Pull request by alouanemed**
#6

### CODE REVIEW

File: test/test_helper.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+require 'state_machines-graphviz'
+require 'minitest/autorun'
+require 'minitest/unit'
+require 'minitest/pride'
+
+require File.expand_path('../../lib/state_machines-graphviz.rb', __FILE__)
```
Comment (seuros) [inline test/test_helper.rb:6]: use `require_relative`.

Comment (alouanemed) [inline test/test_helper.rb:6]: For all of them ?

Comment (seuros) [inline test/test_helper.rb:6]: I don't think this is necessary, line 1 require the gem.

File: test/state_drawing_test.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  def test_should_use_ellipse_shape
+    assert_equal(@node['shape'].to_s.gsub('"', '') ,'ellipse')
+  end
+
+  def test_should_set_width_to_one
+    assert_equal(@node['width'].to_s.gsub('"', '') ,'1')
+  end
+
+  def test_should_set_height_to_one
+    assert_equal(@node['height'].to_s.gsub('"', '') ,'1')
```
Comment (seuros) [inline test/state_drawing_test.rb]: flip assertion 

``` ruby
assert_equal('"1"', @node['height'])
```

File: test/machine_drawing_test.rb [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+  end
+
+  def test_should_raise_exception_if_invalid_option_specified
+    assert_raises ArgumentError, @machine.draw(invalid: true)
+  end
+
+  def test_should_save_file_with_class_name_by_default
+    @machine.draw
+    assert (File.exist?("doc/state_@machines/#{@klass.name}_state.png"))
```
Comment (alouanemed) [inline test/machine_drawing_test.rb]: @seuros

File: test/machine_drawing_test.rb [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+    @machine = StateMachines::Machine.new(@klass, initial: :parked)
+    @machine.event :ignite do
+      transition parked: :idling
+    end
+
+  end
+
+  def test_should_raise_exception_if_invalid_option_specified
+    assert_raises ArgumentError, @machine.draw(invalid: true)
```
Comment (seuros) [inline test/machine_drawing_test.rb]: assert_raises(ArgumentError) { @machine.draw(invalid: true) }

**Pull request closed without merging**